parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoParquetWriter.java

```diff
@@ -81,4 +82,63 @@ public ProtoParquetWriter(Path file, Class<? extends Message> protoMessage) thro
             DEFAULT_BLOCK_SIZE, DEFAULT_PAGE_SIZE);
   }
 
+  /**
+   * Create a new {@link ProtoParquetWriter}.
+   *
+   * @param file                 The file name to write to.
+   * @param protoMessage         Protobuf message class
+   * @param compressionCodecName Compression code to use, or CompressionCodecName.UNCOMPRESSED
+   * @param blockSize            HDFS block size
+   * @param pageSize             See parquet write up. Blocks are subdivided into pages for alignment and other purposes.
+   * @param enableDictionary     Whether to use a dictionary to compress columns.
+   * @param validating           to turn on validation using the schema
+   * @param writeSpecsCompliant  Enable writing specs compliant schemas with parquet-protobuf
+   * @throws IOException if there is an error while writing
+   */
+  public ProtoParquetWriter(Path file, Class<? extends Message> protoMessage,
+          CompressionCodecName compressionCodecName, int blockSize, int pageSize, boolean enableDictionary,
+          boolean validating, boolean writeSpecsCompliant) throws IOException {
+      super(file, new ProtoWriteSupport(protoMessage), compressionCodecName, blockSize, pageSize, pageSize,
+              enableDictionary, validating, DEFAULT_WRITER_VERSION,
+              getConfigWithWriteSpecsCompliant(writeSpecsCompliant));
+  }
+
+  /**
+   * Create a new {@link ProtoParquetWriter}.
+   *
+   * @param file                 The file name to write to.
+   * @param protoMessage         Protobuf message class
+   * @param compressionCodecName Compression code to use, or CompressionCodecName.UNCOMPRESSED
+   * @param blockSize            HDFS block size
+   * @param pageSize             See parquet write up. Blocks are subdivided into pages for alignment and other purposes.
+   * @param writeSpecsCompliant  Enable writing specs compliant schemas with parquet-protobuf
+   * @throws IOException if there is an error while writing
+   */
+  public ProtoParquetWriter(Path file, Class<? extends Message> protoMessage,
+                            CompressionCodecName compressionCodecName, int blockSize,
+                            int pageSize, boolean writeSpecsCompliant) throws IOException {
+    this(file, protoMessage, compressionCodecName, blockSize, pageSize, DEFAULT_IS_DICTIONARY_ENABLED, 
+            DEFAULT_IS_VALIDATING_ENABLED, writeSpecsCompliant);
+  }
+
+  /**
+   * Create a new {@link ProtoParquetWriter}. The default block size is 50 MB.The default
```

Comment:
I believe the default block size is 128MB by default (see https://github.com/apache/parquet-mr/blob/0d55abd05b0e5027c18e60d1ac3b22998dd00951/parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetWriter.java#L41